tests/qasm/test_qasm_to_mpqp.py

```diff
@@ -141,7 +141,7 @@
 
             MyMixedGate q[0], q[1];
             measure q[1] -> c[1];""",
-            [H(0), BasisMeasure([1]), CNOT(0, 1), BasisMeasure([1])],
+            [H(0), BasisMeasure([1], [0]), CNOT(0, 1), BasisMeasure([1], [1])],
```

Comment:
onrgw

Reply:
No, because we are checking the BasisMeasure repr but as it is not associated with a circuit, is not dynamic. So we need to specify the c_targets.